isisd/isisd.c

```diff
@@ -1399,16 +1765,52 @@ DEFUN (show_isis_summary,
 				       vty);
 
 			vty_out(vty, "    IPv6 dst-src route computation:\n");
-			isis_spf_print(area->spftree[SPFTREE_DSTSRC][level-1],
+			isis_spf_print(area->spftree[SPFTREE_DSTSRC][level - 1],
 				       vty);
 		}
 	}
+}
+
+DEFUN(show_isis_summary, show_isis_summary_cmd,
+      "show " PROTO_NAME " [vrf <NAME|all>] summary",
+      SHOW_STR PROTO_HELP VRF_CMD_HELP_STR
+      "All VRFs\n"
+      "summary\n")
+{
+	struct listnode *inode, *nnode;
+	int idx_vrf = 0;
+	struct isis *isis = NULL;
+	char *vrf_name = NULL;
+	bool all_vrf = false;
+
+	ISIS_FIND_VRF_ARGS(argv, argc, idx_vrf, vrf_name, all_vrf)
+	if (!im) {
+		vty_out(vty, PROTO_NAME " is not running\n");
+		return CMD_SUCCESS;
+	}
+	if (vrf_name) {
```

Comment:
again let's fix this pattern in the code base.  I'm going to stop mentioning it, but will expect it to be fixed

Reply:
Sure, will fix it.